docs/design/chart_format.md

```diff
@@ -0,0 +1,664 @@
+
+
+Any short form handler should be able to resolve the default short form as specified above.
+
+
+### Local References
+During chart development, and in other special circumstances, it may be desirable to work with an unversioned, unpackaged local copy of a chart. For the sake of consistency across products, an explicit naming formula is to be followed.
+
+
+In cases where a local path is used in lieu of a full or short name, the path string _must_ begin with either a dot (.) or a slash (/) or the file schema(`file`:).
+
+
+Legal examples of this include:
+```
+  ./foo
+  ../foo
+  /foo
+  //foo
+  /.foo
+  file:///example/foo
+```
+
+Unprefixed relative paths are not valid. For example, `foo/` is not allowed as a local path, as it conflicts with a legal short name, and is thus ambiguous.
+
+
+
+## Appendix A: User Stories for Charts
+Personas:
+
+
+- Operator: Responsible for running an application in production.
+- Service Dev: Responsible for developing new definitions
+- App Dev: Developer who creates applications that make use of existing definitions, but does not create definitions.
+
+
+Stories:
+
+
+- As an operator, I want a build that is 100% reproducible (exact versions)
+- As an app dev, I want to be able to search for definitions...
+        * by keyword, where one app may have multiple keywords (e.g. Redis has storage, message queue)
+        * by name, where name may be "fuzzy"
+        * by author
+        * by last updated date
+- As a service dev, I want a well-defined set of practices to follow
+- As a service dev, I want to be able to work with a team on the same definition
+- As a service dev, I want to be able to indicate when a particular definition is stable, and how stable it is
+- As a service dev, I want to indicate the role I played in building a definition
+- As a service dev, I want to be able to use all of the low-level Kubernetes kinds
+- As an operator, I want to be able to determine how stable a package is
+- As an operator, I want to be able to determine what version of Kubernetes I need to run a definition
+- As an operator, I want to determine whether a definition requires extension kinds (e.g. DaemonSet or something custom), and determine this _before_ I try to install
+- As a service dev, I want to be able to express that my definition depends on others, even to the extent that I specify the version or version range of the other definition upon which I depend
+- As a service dev, I do not want to install additional tooling to write, test, or locally run a definition (this relates to the file format in that the format should not require additional tooling)
```

Comment:
Not sure what this means. They will need to install helm client and server, at a minimum, to test or locally run a definition.